datastore/usermanager.go

```diff
@@ -209,3 +217,32 @@ func rowsToUsers(rows *sql.Rows) ([]User, error) {
 
 	return users, nil
 }
+
+func (db *DB) checkUserPermissions(username string) int {
```

Comment:
This gets changed in the next PR! Can we let this through here and I'll take the naming into account there.